saleor/payment/migrations/0030_payment_partial.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.6 on 2022-01-24 09:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("payment", "0029_alter_payment_options"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="payment",
+            name="partial",
+            field=models.BooleanField(default=False),
+        ),
+    ]
```

Comment:
I believe that we have a huge number of payments on the client's production so I think this migration could produce some downtimes.

Reply:
@cmiacz  What do you think?

Reply:
In PostgreSQL11 adding a field with constant default value is fast no matter how big is the table.
See the docs: https://www.postgresql.org/docs/11/ddl-alter.html#DDL-ALTER-ADDING-A-COLUMN